src/components/dev-dot-content/mdx-components/mdx-alert/mdx-inline-alert.module.css

```diff
@@ -1,3 +1,22 @@
 .spacing {
 	margin: 16px 0;
 }
+
+/** 
+* This components chidlren can render regular markdown
+* that inherits the global long form content type styles
+* however this component needs to maintain specific type
+* styles defined in inline-alert.module.css 
+* 
+* Currently only anchor and paragraph elements are accounted for
+* but depending on authoring behavior, additional adjustments
+* could be made
+**/
+.typographyOverride {
```

Comment:
Ideas of how to circumvent this welcome 🤷🏻

Reply:
See my comments in description: 
> Since these components need to be able to handle markdown syntax, we have to do some extra type style overrides to keep things cohesive if a paragraph or anchor tag renders.

Reply:
🙋🏻 It's not clear to me what cases need this override class in it. Is there an example in the Swingset page where I can manually remove this class name and see broken styles?

Reply:
The swingset page wont parse markdown content that is passed in the same way it does on the long form content pages so its not possible to see this in swingset. The cases where this is needed can be seen in the screenshots in the description. When md syntax is used within the body of the alert, the content will be turned into `p` or `a` tags depending on the markup. This component shouldn't support headings so I've not included those.

Reply:
The easiest way to test this is to seed a local api and database and run the frontend from that local api, its a bit cumbersome. I could take a video if that would be helpful?

Reply:
Interesting point. Where I get stuck is that we should not be adding images + captions to alerts. Ideally inline-alerts should be shorter and be text only. When I look at alerts across Developer they are all using `body-200` except for the [~>](url) alerts of old. Keep me honest though as I could be missing something. Although authors can add `code` elements, `code-blocks`, images and other mdx components into an alert I don't think we should encourage it by making it look intentional if that makes sense. `body-200` in alerts comes straight from HDS and if we are going to divert I want to make sure it is for really good reasons. Open to discuss more though!

Reply:
I also feel like body-300 in alerts feels too strong, with the decreased content width an alert with a paragraph of plain text takes up the entire screen and blows out the hierarchy imo.
<img width="659" alt="image" src="https://user-images.githubusercontent.com/81585949/211883750-b8cf9bc3-2d3a-4315-995c-1d4ce9557c64.png">

Reply:
I added some examples in the authoring docs of the cases where these styles are needed, mainly when mdx syntax is used instead of plain strings.

Reply:
Applying the body-200 styles to plain text alerts would definitely make things simpler, but then the alert title needs to be adjusted so it doesn't look out of balance. 

It's tricky because the ideal use case is just the short plain text string, but we need this type override anytime md syntax is used. So for example if there is a link in the body, but it is still a short string, a `p` and `a` tag will be generated that the larger styles apply.

Reply:
Also when I dig through the content with the old syntax, there are soooo many examples of varied content being used in these alerts. Again I don't want to encourage the use this way but we need to be realistic for how authors will actually use these.

Reply:
☝️ I think that's mainly my concern - I'm a little worried it might not be realistic to think we'll be able to fully transition away from images, lists, and other varied content within alerts. If that's the case, then I think that varied content within alerts might be something we want to dive into a little deeper. Less of an immediate blocking concern, and more of a future thought!

💭  Related thought: if we want to avoid varied content in these new alerts, and are strongly aiming for alerts to be text-only (but allowing bold, italics, links, and other "inline" markdown), would it make sense for the new MDX component prevent use of anything but `<p />` children? This could similar to how eg. `CodeBlockConfig` or `ImageConfig` throws an error for unexpected children. (This might not be a great idea if we're aiming to really push for migrating old alert sigils to the new MDX syntax... or if it just feels too restrictive generally... but it might be a possibility if we're aiming to refactor content to meet a new "alerts should have text only" authoring requirement).

Reply:
Good idea @zchsh, something @braydenlove and I have been chatting through. I think I don't know enough about the authors needs to feel good about putting hard restrictions on this component like that. But I guess we should do whatever will give us the least headache later on. If we leave it open for authors and then come back to adjust styles for the varied content situations based on real world use cases, then it could be a better time to make those calls.

If we restrict this new component but we still support the old one (even though we call it deprecated), and if the new one is too restrictive so authors continue to use the old syntax, that would be not good. So imo we should leave this open, but encourage folks to not use images in them. And if we need to adjust styles later on based on how they're actually using it, I think thats ok.

Reply:
💯 I dig the above logic, +1 on not being too restrictive. Thanks y'all for the extra context 🙇

Reply:
I think the easiest path forward is to start flexible so we're able to support existing content. Then when content-conformance checks are stable, we can add a rule for what kind of content is allowed inside of these alerts. 😊